app/assets/src/components/views/discovery/DiscoverySidebar/ProjectDescription.jsx

```diff
@@ -53,6 +53,7 @@ class ProjectDescription extends React.Component {
         },
         () => {
           this._save(this.props.project.id, this.state.description);
+          this.props.onProjectDescriptionSave(this.state.description);
```

Comment:
nit: Although this a simple function and the pattern was not followed before here, we try to use a pattern of loading `props` and `state` at the top of the function like:
```javascript
const { changed } = this.state;
const { onProjectDescriptionSave } = this.props;
```
Do you mind applying this here and the other functions you touched?

Reply:
Got it! I will change rn

Reply:
fixed!